docs/source/conf.py

```diff
@@ -89,3 +89,30 @@
     }
 ]
 
+# Generate PDF from pds-api-specification.md URL
+
+from markdown import markdown
+import pdfkit
+import requests
+
+markdown_filename = '../build/search-api-user-guide/pds-api-specification.md'
+html_filename = '../build/search-api-user-guide/pds-api-specification.html'
+pdf_filename = '../build/search-api-user-guide/pds-api-specification.pdf'
+rst_filename = 'search-api-user-guide/pds-api-specification.rst'
+
+url = 'https://raw.githubusercontent.com/NASA-PDS/pds-api/main/docs/spec/pds-api-specification.md'
+
+r = requests.get(url, allow_redirects=True)
+open(markdown_filename, 'wb').write(r.content)
+
+with open(markdown_filename, 'r') as f:
+    html_text = markdown(f.read(), output_format='html4')
```

Comment:
@ramesh-maddegoda should this be `output_format='html'` instead of `html4`?

Reply:
It seems the previous reference of the markdown library I checked had `html4`. The latest https://python-markdown.github.io/reference/ has `html` as follows in output formats.

```
output_format:
   Format of output.

    Supported formats are:

    "xhtml": Outputs XHTML style tags. Default.
    "html5": Outputs HTML style tags.

The values can be in either lowercase or uppercase.
```

However, I changed the output_format to `html` to avoid this sonatype-lift issue.